tests/foreman/cli/test_host_collection.py

```diff
@@ -114,18 +114,18 @@ def test_positive_create_with_limit(self):
         for limit in ('1', '3', '5', '10', '20'):
             with self.subTest(limit):
                 new_host_col = self._new_host_collection(
-                    {'max-content-hosts': limit})
+                    {'max-hosts': limit})
                 self.assertEqual(new_host_col['limit'], str(limit))
 
     @skip_if_bug_open('bugzilla', 1214675)
     @tier1
     def test_positive_create_with_unlimited_chosts(self):
```

Comment:
You know that this is out of context for this PR, right? :) I'll change it though